test/controllers/CreditsSummaryControllerSpec.scala

```diff
@@ -97,6 +116,66 @@ class CreditsSummaryControllerSpec extends TestSupport with MockCalculationServi
       }
     }
 
+    "all calls are returned correctly and Referer was a Credit and Refund page" should {
+      "show the Credits Summary Page and back link should be to the Credit and Refund page" in {
+        val chargesList = creditAndRefundCreditDetailListMFA
+
+        enable(MFACreditsAndDebits)
+        mockSingleBusinessIncomeSource()
+        mockCreditHistoryService(chargesList)
+        setupMockAuthRetrievalSuccess(testAuthSuccessWithSaUtrResponse())
+
+        when(mockCreditService.getCreditCharges()(any(), any()))
+          .thenReturn(Future.successful(List(financialDetailCreditAndRefundCharge)))
+
+        val expectedContent: String = creditsSummaryView(
+          backUrl = creditAndRefundUrl,
+          utr = Some(testSaUtrId),
+          enableMfaCreditsAndDebits = true,
+          charges = chargesList,
+          maybeBalanceDetails = Some(financialDetailCreditCharge.balanceDetails),
+          calendarYear = testTaxYear
```

Comment:
this is still creating a confusion and would be a source of confusion in the future; 
pls set variable as calendarYear across this unit test file

Reply:
Thank you, I've just pushed the fix